advocacy_docs/supported-open-source/barman/pg-backup-api/03-tasks.mdx

```diff
@@ -0,0 +1,83 @@
+---
+title: 'Operations'
+description: 'Instructions for creating and querying tasks operations on the Postgres Backup API'
+tags:
+    - barman
+    - backup
+    - recovery
+    - postgresql
+    - pg-backup-api
+---
+
+### Prerequisites
+
+ You can ask Barman to perform operations on upstream servers. If you plan to use that feature, the following setup is expected:
+
+ * Postgres backup API (pg-backup-api) must be installed on the same server as Barman. Please check [How to install Postgres backup API](01-installation) if you haven't done it already.
+ * SSH passwordless login must be in place among the different hosts.
+ * Ownership of destination directory or write permissions must be granted to the user Barman is going to use.
+ * For security reasons, the API will listen to on localhost only. You need to use some proxy to forward the traffic. The Apache http server was used during our tests. Please [read our notes about how to define a virtual host](02-securing-pg-backup-api#adding-a-virtualhost-definition-for-postgres-backup-api).
+
```

Comment:
Shouldn't we have a section "Available operations"? Even if we have one or two, we should have this as a section. The first thing I want to find is which operations are available

Reply:
done